rally_ovs/plugins/ovs/scenarios/ovn_network.py

```diff
@@ -55,24 +58,64 @@ def create_routers_bind_ports(self, router_create_args=None,
                                   port_create_args=None,
                                   ports_per_network=None,
                                   port_bind_args=None):
-
+        overlay_setup = router_create_args.get("overlay", True)
```

Comment:
It seems unnecessary for a new argument for overlay_setup, since routers are already about overlay.